search-handlers/src/test/java/no/unit/nva/search/ResourceSearchContainerTests.java

```diff
@@ -66,14 +69,20 @@ public class ResourceSearchContainerTests {
   public static void beforeAll() {
     container.withEnv("indices.query.bool.max_clause_count", "2048").start();
 
-    var restClientBuilder = RestClient.builder(HttpHost.create(container.getHttpHostAddress()));
-    var restHighLevelClientWrapper = new RestHighLevelClientWrapper(restClientBuilder);
-    var cachedJwtProvider = setupMockedCachedJwtProvider();
-    indexingClient = new IndexingClient(restHighLevelClientWrapper, cachedJwtProvider);
+    RestClientBuilder restClientBuilder = null;
+    try {
+      restClientBuilder = RestClient.builder(HttpHost.create(container.getHttpHostAddress()));
```

Comment:
Same here, could these be merged?

Also, since this is a re-used try/catch, would it make sense to extract some of it (at least the part that throws exceptions) to a separate, shared method?